doc/documents/Examples_Tutorials/Examples_Tutorials.rst

```diff
@@ -0,0 +1,623 @@
+
+Consider a small example not directly related to the CIFAR-10: 
+ 
+.. table:: Integer Bits Derivation Considering Accumulator Restrictions
+   :widths: 30, 30, 20, 20
+   :align: center
+   
+   +------------------------------+-----------------------+-------------------+------------------+
+   |                              |                       | **Layer X Conv**  | **Layer X+1 FC** |
+   +------------------------------+-----------------------+-------------------+------------------+
+   |                              | Layer Input Bits      |         5         |       5          |
+   | **Integer Bit Requirements** +-----------------------+-------------------+------------------+
+   | *(fx8 operands)*             | Layer Weight Bits     |        -1         |      -1          |
+   |                              +-----------------------+-------------------+------------------+
+   |                              | Layer Output Bits     |         5         |       5          |
+   +------------------------------+-----------------------+-------------------+------------------+
+   |                              | MACs per Output value | 801 (32*5*5+1)    | 1025 (64*16+1)   | 
+   |                              +-----------------------+-------------------+------------------+
+   | **Accumulator Restrictions** | Required Extra Bits   |        10         |      11          |   
+   |                              +-----------------------+-------------------+------------------+   
+   |                              | Not Enough Bits       |     10 – 9 = 1    |   11 – 9 = 2     |   
+   +------------------------------+-----------------------+-------------------+------------------+  
+   |                              | Layer Input Bits      |      5 + 1 = 6    |    5 + 1 = 6     |
+   |                              | (updated)             |                   |                  |
+   | **Integer Bit Requirements** +-----------------------+-------------------+------------------+
+   | *(fx16 operands)*            | Layer Weight Bits     |       -1          |   -1 + 1 = 0     |
+   |                              | (updated)             |                   |                  |
+   |                              +-----------------------+-------------------+------------------+
+   |                              | Layer Output bits     | 6 (next layer in) |       5          |
+   |                              | (updated)             |                   |                  |
+   +------------------------------+-----------------------+-------------------+------------------+
+   
+   
+   
+   
+   
+   
+Ensure that you follow these steps:
```

Comment:
We are describing what have been calculated in the table above. I don;t think "Ensure" is a right word.  Can this and list below be converted into description form, not "to-do" list?